start.py

```diff
@@ -169,6 +185,19 @@ def main(args):
         for node in nodes:
             node.volumes.append(sdc_parcel_image)
 
+        if args.navigator:
+            try:
+                navigator_stage_lib = _get_navigator_stage_lib_name(args.cm_version,
+                                                                    args.additional_stage_libs_version)
+                stage_lib_name = '{}/{}/additional-datacollector-libs:{}'
+                navigator_stage_lib_image = stage_lib_name.format(args.registry,
+                                                                  args.namespace or DEFAULT_NAMESPACE,
+                                                                  navigator_stage_lib)
+                logger.info('Adding stage lib image for Navigator %s to primary node ... ', navigator_stage_lib_image)
+                primary_node.volumes.append(navigator_stage_lib_image)
+            except:
+                raise Exception('No Navigator stage library found for CM version %s.', args.cm_version)
```

Comment:
You can't use Exception like this (I think you're thinking of the `logging` modules). Gotta do `Exception('...'.format())`

Reply:
Done.